pages/en/learn/modules/publishing-node-api-modules.md

```diff
@@ -45,7 +45,7 @@ will look like this:
 > reason for this is that the tag refers to exactly one version. So, if the
 > package maintainer chooses to tag a later version of the package using the
 > same tag, `npm update` will receive the later version. This should be acceptable
-> given the currently experimental nature of N-API. To depend on an N-API-enabled
+> given the currently experimental nature of Node-API. To depend on an Node-API-enabled
```

Comment:
[Node-API is now stable](https://nodejs.org/api/n-api.html#node-api) - can you deop this sentence?

Reply:
i made the requested change